src/bin/index.ts

```diff
@@ -100,8 +101,8 @@ const main = async () => {
   let localeFiles: TranslationFile[] = [];
 
   const pattern = isMultiFolders
-    ? `{${localePath.join(",").trim()}}/**/*.json`
-    : `${localePath.join(",").trim()}/**/*.json`;
+    ? `{${localePath.join(",").trim()}}/**/*.{json,yaml,yml}`
+    : `${localePath.join(",").trim()}/**/*.{json,yaml,yml}`;
```

Comment:
@busypeoples @steos I went with default support of YAML files, but maybe you would prefer using a command argument to choose the format

Reply:
@antoinm I think we should go for the default approach without having to specify a command argument.